### ISSUE TITLE
AttributeError: 'module' object has no attribute 'table_to_hdu'

### ISSUE THREAD

**Issue by xihou** [fermiPy/fermipy#194]
Hi,

I encountered this problem when writing the fit result to a fits file. It has been seen a few months ago and has been fixed, something related to astropy. But now it appears again. I have the develop version installed: 
fermipy-0.16.0+126.gdec7

Thanks!
Xian

**Comment by woodmd** [comment]
You're likely seeing this error because you're using the anaconda python distribution installed in GLAST_EXT.  The version of astropy in this distribution is too old for fermipy.  Going forward probably the best solution is that you just create your own anaconda distribution so that you can have full control of package versions.  I've discussed with @dimauromattia and we plan to update the documentation in the next few weeks with some instructions for creating your own anaconda installation.  In the meantime you can have a look at this [page](https://github.com/fermiPy/fermipy/wiki/Creating-an-Anaconda-Python-Installation-at-SLAC).

**Comment by xihou** [comment]
By the way, can't we simply update the astropy version in GLAST_EXT to solve the problem?

**Comment by me-manu** [comment]
Hi @xihou, 
updating the astropy version of the science tools is out of our hands I'm afraid. At the step on this [page](https://github.com/fermiPy/fermipy/wiki/Creating-an-Anaconda-Python-Installation-at-SLAC) where you export the installation directory, can't you choose a different directory then $HOME/anaconda? The installation takes quite some space, so I would recommend to install it on your /nfs/farm/g/glast/u/your-user-name directory, so for example:
```export CONDABASE=/nfs/farm/g/glast/u/your-user-name directory/anaconda```

**Comment by xihou** [comment]
Hi，thanks for the suggestions! Installing anaconda in my nfs disk works and I can run fermipy without errors. But how about the dmpipe? I tried also installing it in my afs disk, but not worked. I always have the error "sh: dmpipe-calc-sed-limits: command not found". I have used below commands following the online dmpipe instruction:
git clone https://github.com/fermiPy/dmpipe.git
cd dmpipe
#To install the current commit
python setup.py install --user

Would you pls help to debug the dmpipe as well? I need it to convert SED to sigmav ul.

Thanks!

**Comment by me-manu** [comment]
Are you sure that you are using the anaconda python version when you try to install dmpipe? When you type ```which python``` into the terminal, does it give you the right path to the anaconda version?

**Comment by xihou** [comment]
I do have the anaconda python version if type 'which python'. I finally
found that the problem comes from the --user argument when installing
dmpipe. This will install some dependencies always into my HOME .local, but
not my nfs disk. Simply using 'python install setup.py' resolved the
problem.

Then I did a quick test for the MAX, MED and MIN maps (currently for only 1
MC run), for MAX, I got again this error which has appeared before and
seems related to the likelihood profile range, as Matt has suggested. Could
this be fixed?  Thanks!

  File
"/nfs/farm/g/glast/u/hou/anaconda/lib/python2.7/site-packages/fermipy/castro.py",
line 203, in _compute_mle
    while np.sign(self._interp.derivative(self._interp.x[ix0])) == \
IndexError: index 200 is out of bounds for axis 0 with size 200

2018-03-09 2:24 GMT+08:00 me-manu <notifications@github.com>:
